1_k/4_imp++/README.md

```diff
@@ -14,7 +14,7 @@ anywhere a statement can occur.  In this tutorial we will learn the following:
 * How to refine configurations as a language evolves.
 * How to define and use fresh elements of desired sorts.
 * How to tag syntactic constructs and rules, and how to use such tags
-  with the `superheat`/`supercool`/`transition` options of `kompile`.
+  with the `superheat`/`supercool` options of `kompile`.
```

Comment:
I think these options are deprecated as well; I presume that they're part of the same deprecated handling of transition semantics in the Java backend? A grep for `super(heat|cool)` in the K repo only turns up documentation references.